Make link text more descriptive

RobertoJBeltran opened the pull request:
Make link text more descriptive for improved accessibility.

<!-- Please follow this checklist and put an x in each of the boxes, like this: [x]. It will ensure that our team takes your pull request seriously. -->

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

<!--If your pull request closes a GitHub issue, replace the XXXXX below with the issue number.-->

Closes #XXXXX

RobertoJBeltran committed:
Make link text more descriptive

Make link text more descriptive for improved accessibility.

theNaut1lus reviewed (changes requested):
Add MDN in link title to make the audience know where they are going when they click on it.
so basically from
`click this link to know more` -> `click this link to see the MDN documentation`

something along the lines of this

RobertoJBeltran committed:
Add mention of MDN to link description

As suggested by Sidak Singh Aulakh, added reference to MDN web docs which is the link's destination.

RobertoJBeltran commented:
I made a new commit per your suggestion

RandellDawson commented on curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md:
```diff
@@ -12,7 +12,7 @@ Here's how you would rewrite and emphasize the text of our heading:
 jQuery also has a similar function called <code>.text()</code> that only alters text without adding tags. In other words, this function will not evaluate any HTML tags passed to it, but will instead treat it as the text you want to replace the existing content with.
 Change the button with id <code>target4</code> by emphasizing its text.
-Check this <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">link</a> to know more on the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62;</code> and their uses.
+Check this link to <a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">go to the MDN web docs for <code>&#60;em&#62;</code></a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
```
@RobertoJBeltran You actually did not make the change @co16353sidak suggested.  Also, after reviewing this myself, I suggest the following change:
```suggestion
<a href="https://developer.mozilla.org/en/docs/Web/HTML/Element/em" target="_blank">View the MDN web docs for &#60;em&#62;</a> to learn the difference between <code>&#60;i&#62;</code> and <code>&#60;em&#62</code> and their uses.
```

co16353sidak replied:
@RobertoJBeltran I agree with @RandellDawson here, this suggestion seems much better to me

RandellDawson reviewed (changes requested):
@RobertoJBeltran Please see my suggested changes.

If you have any questions, please let us know.

RandellDawson committed:
Update curriculum/challenges/english/03-front-end-libraries/jquery/change-text-inside-an-element-using-jquery.english.md

Adjust link wording

Co-Authored-By: RobertoJBeltran <45843750+RobertoJBeltran@users.noreply.github.com>

RobertoJBeltran commented:
@RandellDawson  Understood, I will do better next time

theNaut1lus reviewed (approved):
@RandellDawson after the latest commit, everything Is LGTM 🎉 
Ready to be merged

Manish-Giri reviewed (approved):
Hi @RobertoJBeltran 

Congrats on your first pull request (PR)! 🎉

Thank you for your contribution to the page! 👍 We're happy to accept these changes and look forward to future contributions. 📝

RobertoJBeltran commented:
@Manish-Giri Thank you, I look forward to making more contributions
